.github/workflows/kafka_demo_test.yml

```diff
@@ -0,0 +1,121 @@
+name: Kafka Demo Test 
+
+on:
+  pull_request:
+    branches: 
+      - main
+  workflow_call:
+    inputs:
+      sidecar-registry:
+        description: "The registry to get the image from"
+        required: true
+        default: "mcr.microsoft.com"
+        type: string
+      key-release-image:
+        description: "The image of the SKR sidecar to use"
+        required: true
+        default: "aci/skr:2.7"
+        type: string
+      consumer-image: 
```

Comment:
This and the producer should have a default here since this is the main test file. That way if we just want to run the pipeline to test SKR, we don't have to specify these images if they haven't changed (and they shouldn't change much).

Reply:
Actually, we probably don't need workflow_call here since this file isn't being called by any other yamls.